language-support/ts/daml-ledger/index.ts

```diff
@@ -220,6 +220,12 @@ type LedgerOptions = {
    * as it is the case with the development server of `create-react-app`.
    */
   wsBaseUrl?: string;
+  /**
+   * Optional number of milliseconds a connection has to be live to be considered healthy. If the
+   * connection is closed after being live for at least this amount of time, the `Ledger` tries to
+   * reconnect, else not.
+   */
+  wsMinLiveTime?: number;
```

Comment:
What do you think about `reconnectThreshold` as the name for this?

Reply:
that sounds fine to me. I'll change it.